routes/web.php

```diff
@@ -12,7 +17,45 @@
 | contains the "web" middleware group. Now create something great!
 |
 */
+//register
+Route::get('/', [RegisterController::class, 'index'])->middleware('guest')->name('home');
+Route::get('/register', [RegisterController::class, 'create'])->middleware('guest');
+Route::post('register', [RegisterController::class, 'store'])->middleware('guest');
```

Comment:
`/register`

and below 

`register`

Have single way for all the routes below